hil/ext/network_allocators/vlan_pool.py

```diff
@@ -69,6 +69,20 @@ def validate_network_id(self, net_id):
         except ValueError:
             return False
 
+    def network_id_available(self, net_id):
```

Comment:
> Another way to do this would be to have a claim_network_id, which raised an AllocationError if the id is unavailable.

I like this, I am gonna do that.